src/components/Form/Form.jsx

```diff
@@ -4,29 +4,26 @@ import { validateEmail, validateCPF } from './FormValidation.jsx'
 import { useState } from 'react'
 
 export const Form = () => {
-    const [input, setInput] = useState({})
+    const [email, setEmail] = useState("")
+    const [CPF, setCPF] = useState("")
```

Comment:
É interessante seguirmos um padrão quanto a nomes iniciando com letras maiúsculas e minúsculas, mesmo quando se tratar de siglas. Geralmente variáveis com todas as letras maiúsculas são [valores imutáveis](https://www.robinwieruch.de/javascript-naming-conventions/#:~:text=There%20are%20no%20special%20naming,UPPERCASE%20if%20it%20is%20immutable.). 
```suggestion
    const [cpf, setCPF] = useState("")
```